app/components/primer/component.rb

```diff
@@ -5,7 +5,7 @@
 module Primer
   # @private
   class Component < ViewComponent::Base
-    include ViewComponent::SlotableV2 unless ViewComponent::VERSION::STRING.to_f >= 2.28
+    include ViewComponent::SlotableV2 unless ViewComponent::Base < ViewComponent::SlotableV2
```

Comment:
**I think** this is including the deprecated slotable v2 API if the consumer's view component version still supports it but I'm not sure. Mind elaborating why we need to do this? Do any of our components use the deprecated `ViewComponent::SlotableV2`?

Reply:
the idea is to only include it if `ViewComponent::Base` doesn't
we just don't want to include it twice to avoid bloating